extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -505,5 +511,6 @@ export function installUserNotificationManagerForTesting(ampdoc) {
 // Register the extension services.
 AMP.extension(TAG, '0.1', function(AMP) {
   AMP.registerServiceForDoc(SERVICE_ID, UserNotificationManager);
+  AMP.registerServiceForDoc(NOTIFICATION_UI_MANAGER, NotificationUiManager);
```

Comment:
I am a bit concerned that both components register the notificationUiManager service.